apps/api/src/app/events/usecases/process-subscriber/process-subscriber.usecase.ts

```diff
@@ -34,11 +37,27 @@ export class ProcessSubscriber {
 
     const notification = await this.createNotification(command, template._id, subscriber);
```

Comment:
A small suggestion, I would add an API E2E tests as early as now so we can begin to create the expected behaviours and work against a running tests suite rather than manually executing. In this case we will have a good behavioural check history rather than creating them all at the end :)

Reply:
Yeah just liked us to agree that we are agreeing on that this is how we should do it :)